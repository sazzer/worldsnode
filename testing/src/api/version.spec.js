// @flow

import { request } from '../request';

test('/api/version', async () => {
    const response = await request('/api/version');
    expect(response.status).toEqual(200);
    expect(response.data).toMatchSnapshot();
});
