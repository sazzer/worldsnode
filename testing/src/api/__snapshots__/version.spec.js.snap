// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`/api/version 1`] = `
Object {
  "version": "1.0.0",
}
`;
